crates/wasmtime/src/runtime/vm/byte_count.rs

```diff
@@ -94,6 +94,9 @@ impl HostAlignedByteCount {
             .ok_or(ByteCountOutOfBounds(ByteCountOutOfBoundsKind::Add))
     }
 
+    // Note: saturating_add should not be added! usize::MAX is not a power of 2
+    // so is not aligned.
```

Comment:
Hm I guess it can be added but it would have to be up to the last representable page, not `usize::MAX`. Haven't seen a good use case for it though.